cmd/deployment.go

```diff
@@ -32,6 +35,9 @@ var (
 
 # Create new deployment with Kubernetes executor.
   $ astro deployment create new-deployment-name-k8s --executor=k8s
+
+# Create new deployment with Kubernetes executor.
+  $ astro deployment create new-deployment-1-10-10-airflow-k8s --executor=k8s --airflowVersion=1.10.10
```

Comment:
@andriisoldatenko I'm thinking out loud but I almost feel like this "deployment name" might confuse the user and make them think that what they name their deployment affects whether or not these flags are successfully passed.

Should we just do this as the example:

`$ astro deployment create my-new-deployment --executor=k8s --airflowVersion=1.10.10`

WDYT?

Reply:
fixed in 4ece5bd962c8dd565dbf0215ef45af1f87576765.